README.md

```diff
@@ -13,19 +13,26 @@ located.
 const path = require('path');
 
 const nconf = require('nconf');
-require('./lib/nconfetti');
 
-const path = path.resolve(__dirname, 'tests/configs/env');
+require('./../index.js'); // replace with require('nconfetti');
 
 nconf.argv()
   .env('__');
 
-nconf.use('Nconfetti', {path: path, env: process.env.NODE_ENV});
+nconf.use('Nconfetti', {path: path.resolve(__dirname, '../tests/configs/without_env')});
 
-console.log(nconf.get('development_config:development_config'));
+console.log(nconf.get('simple_config:entry'));
+```
 
+## Try a sample
 
-```
+- see sample/simple.js
+- runit with
+  - npm install
+  - node samples/simple.js
+ - have fun
+
+## Description
 
 After require `nconfetti` the backend will be added as storage backend
 for `nconf`. Then it needs to be configired with `path`and `env`.
```

Comment:
Ich würde das aber auch nicht als sample in Ordnern verstecken, sondern direkt in die README schreiben. Ich bin immer dankbar wenn ich fremde READMEs lese und dort alles finde, anstatt mich erst durch das gesamte Repo zu arbeiten und mir dort Beispiele zu suchen.